packages/cmf/src/reducers/settingsReducers.js

```diff
@@ -75,9 +75,8 @@ export function settingsReducers(state = defaultState, action) {
 		case ACTIONS.REQUEST_OK:
 			return Object.assign({}, state, { initialized: true }, prepareSettings(action.settings));
 		case ACTIONS.REQUEST_KO:
-			alert(`Settings can't be loaded ${get(action, 'error.message')}`); // eslint-disable-line
```

Comment:
I think the `Settings can't be loaded ${get(action, 'error.message')}` should be ported into the console.error to better help the user to pinpoint the issue